### PR TITLE
partial mocked download dialog

### PR DESCRIPTION
**Pull request by edewit**
marvelapp: https://marvelapp.com/prototype/908c3c3/screen/72040454

### CODE REVIEW

**Comment by edewit** [comment]
partial mocked because it's depended on #138

File: src/components/download-dialog/DownloadDialog.tsx [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+  Form,
+  FormGroup,
+  Select,
+  SelectOption,
+  SelectVariant,
+  Stack,
+  StackItem,
+  TextArea,
+} from "@patternfly/react-core";
+// import serverInfo from "../../clients/__tests__/mock-serverinfo.json";
```
Comment (ssilvert) [inline src/components/download-dialog/DownloadDialog.tsx:14]: Remove commented line

**Review by ssilvert — COMMENTED** [review]
@edewit I take it that you did it this way because we don't have ServerInfo yet?  We just need to log an issue to come back to this once ServerInfo is implemented.

If your answer is yes, we can merge this when the issue is created.

**Comment by edewit** [comment]
@ssilvert no there is already a PR for this like I mentioned #138

**Review by ssilvert — APPROVED** [review]

**Pull request merged**